script/dogs.js

```diff
@@ -0,0 +1,3 @@
+$('.flex-dog').hover(function(e) {
+  $('.flex-dog').addClass('dog-border');
```

Comment:
This is close, but would be nice if this file was on the index page as well. I had to dig to figure out why nothing was happening and found this was only on the dogs page.

Also, see if you can find a way to add a class (or even better toggle the class) on an individual tile.